packages/@sanity/default-login/src/LoginWrapper.js

```diff
@@ -24,11 +24,12 @@ export default class LoginWrapper extends React.PureComponent {
     ]).isRequired,
     title: PropTypes.node,
     description: PropTypes.node,
-    sanityLogo: PropTypes.oneOfType([
+    sanityLogo: PropTypes.node,
+    SanityLogo: PropTypes.oneOfType([
       PropTypes.node,
```

Comment:
The props for SanityLogo should now be PropTypes.func right?

Reply:
yes